specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/HybridLinks.json

```diff
@@ -0,0 +1,329 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-11-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/hybridLink": {
+      "get": {
+        "tags": [
+          "HybridLinks"
+        ],
+        "description": "Gets a list of hybrid links in instance.",
+        "operationId": "HybridLinks_ListByInstance",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of hybrid links.",
+            "schema": {
+              "$ref": "#/definitions/HybridLinkListResult"
+            }
+          },
+          "default": {
```

Comment:
>default": { [](start = 11, length = 11)

Please add the schema for error. The same comment applies to other operations as well.